doc/modules/homology.rst

```diff
@@ -15,3 +15,4 @@
    homology.SparseRipsPersistence
    homology.EuclideanCechPersistence
    homology.CubicalPersistence
+   homology.FlagserPersistence
```

Comment:
I don't know if this makes any difference but should this maybe be preceding `CubicalPersistence` (because it is simplicial homology)? Will it impact the order of display?

Reply:
Yes, it impacts the order of display. Fair point.